src/sdk/base/EtherspotWalletAPI.ts

```diff
@@ -69,8 +91,178 @@ export class EtherspotWalletAPI extends BaseAccountAPI {
     return accountContract.interface.encodeFunctionData('uninstallModule', [moduleTypeId, module, deinitData]);
   }
 
+  // Function to get all executors
+  async getAllExecutors(): Promise<string[]> {
+    // Assuming getExecutorsPaginated is a contract function that can be called
+    // Initialize variables
+    let pageSize = 50;
```

Comment:
Should we pass this as a configurable parameter (with default value as 50)?

Reply:
updated with usage of DEFAULT_PAGE_SIZE and caller can override with the value they wish to paginate with